civihr_default_theme/assets/css/civihr_default_theme.style.css

```diff
@@ -1,4 +1,4 @@
n--download,[data-ssp-upload-document] .input-group-sm>.input-group-btn>textarea.chr_action--icon--edit,[data-ssp-upload-document] .input-group-sm>.input-group-btn>textarea.chr_action--icon--list,[data-ssp-upload-document] .input-group-sm>.input-group-btn>textarea.chr_action--icon--print,select[multiple].input-sm,.input-group-sm>select.form-control[multiple],.ui-datepicker .ui-datepicker-header .input-group-sm>select[multiple],.input-group-sm>select.input-group-addon[multiple],.input-group-sm>.input-group-btn>select.btn[multiple],.input-group-sm>.input-group-btn>select.chr_action[multiple],.input-group-sm>.input-group-btn>select.chr_action--icon[multiple],.input-group-sm>.input-group-btn>select.chr_action--icon--calendar[multiple],.input-group-sm>.input-group-btn>select.chr_action--icon--download[multiple],.input-group-sm>.input-group-btn>select.chr_action--icon--edit[multiple],.input-group-sm>.input-group-btn>select.chr_action--icon--list[multiple],.input-group-sm>.input-group-btn>select.chr_action--icon--print[multiple],[data-ssp-upload-document] .input-group-sm>select.form-control[multiple],.ui-datepicker .ui-datepicker-header [data-ssp-upload-document] .input-group-sm>select[multiple],[data-ssp-upload-document] .input-group-sm>select.input-group-addon[multiple],[data-ssp-upload-document] .input-group-sm>.input-group-btn>select.btn[multiple],[data-ssp-upload-document] .input-group-sm>.input-group-btn>select.chr_action[multiple],[data-ssp-upload-document] .input-group-sm>.input-group-btn>select.chr_action--icon[multiple],[data-ssp-upload-document] .input-group-sm>.input-group-btn>select.chr_action--icon--calendar[multiple],[data-ssp-upload-document] .input-group-sm>.input-group-btn>select.chr_action--icon--download[multiple],[data-ssp-upload-document] .input-group-sm>.input-group-btn>select.chr_action--icon--edit[multiple],[data-ssp-upload-document] .input-group-sm>.input-group-btn>select.chr_action--icon--list[multiple],[data-ssp-upload-document] .input-group-sm>.input-group-btn>select.chr_action--icon--print[multiple]{height:auto}.form-group-sm .form-control,.form-group-sm .ui-datepicker .ui-datepicker-header select,.ui-datepicker .ui-datepicker-header .form-group-sm select{height:30px;padding:5px 10px;font-size:12px;line-height:1.5;border-radius:0}.form-group-sm select.form-control,.form-group-sm .ui-datepicker .ui-datepicker-header select,.ui-datepicker .ui-datepicker-header .form-group-sm select{height:30px;line-height:30px}.form-group-sm textarea.form-control,.form-group-sm select[multiple].form-control,.form-group-sm .ui-datepicker .ui-datepicker-header select[multiple],.ui-datepicker .ui-datepicker-header .form-group-sm select[multiple]{height:auto}.form-group-sm .form-control-static{height:30px;min-height:32px;padding:6px 10px;font-size:12px;line-height:1.5}.input-lg,.input-group-lg>.form-control,.ui-datepicker .ui-datepicker-header .input-group-lg>select,.input-group-lg>.input-group-addon,.input-group-lg>.input-group-btn>.btn,.input-g
```

Comment:
I have a feeling that something may be wrong here, it has 2 rules for `border-color`, could you please doublecheck that the dist file changes are expected?:

![image](https://user-images.githubusercontent.com/3973243/47786897-23cccf00-dd05-11e8-8a21-e7a453c45c10.png)

Reply:
I know this is not due to your change, but because I can now spot it, I decided to raise it.

Reply:
I was asking about duplicating the same style when there is a non-important rule and the same, important rule. I guess that's just how the compiler works, it simply does not care.

However, the change you have just introduced requires Backstop tests I think.

Reply:
The reply to this comment is here: https://github.com/compucorp/org.civicrm.shoreditch/pull/135#discussion_r229676147